tests/testcase.coffee

```diff
@@ -25,11 +25,22 @@ class TestCase
     keyBindings = new KeyBindings @settings
     @keyhandler = new KeyHandler @view, keyBindings
     @register = @view.register
+    @name = options.name || "an anonymous test"
+    unless callback then throw "No callback in test"
+    if it?
```

Comment:
i'm guessing that's also why you wouldn't just do

it 'describe the test, () ->
  t = new TestCase [ ... ]
  t.sendKeys ... 

is that right?